# Make the server depend on libdrm-poulsbo instead of libdrm so that the 
# emenlow libdrm can coexist with the normal libdrm.
# jdike@linux.intel.com

Index: xorg-server-1.7.99.2/configure.ac
===================================================================
--- xorg-server-1.7.99.2.orig/configure.ac
+++ xorg-server-1.7.99.2/configure.ac
@@ -763,7 +763,7 @@ dnl List of libraries that require a spe
 LIBAPPLEWM="applewm >= 1.4"
 LIBDMX="dmx >= 1.0.99.1"
 LIBDRI="dri >= 7.1.0"
-LIBDRM="libdrm >= 2.3.0"
+LIBDRM="libdrm_poulsbo >= 2.3.0"
 LIBGL="gl >= 7.1.0"
 LIBXEXT="xext >= 1.0.99.4"
 LIBXI="xi >= 1.2.99.1"
@@ -2002,7 +2002,7 @@ if test "$KDRIVE" = yes; then
         XEPHYR_REQUIRED_LIBS="$XEPHYR_REQUIRED_LIBS xv"
     fi
     if test "x$DRI" = xyes && test "x$GLX" = xyes; then
-        XEPHYR_REQUIRED_LIBS="$XEPHYR_REQUIRED_LIBS $LIBGL libdrm"
+        XEPHYR_REQUIRED_LIBS="$XEPHYR_REQUIRED_LIBS $LIBGL libdrm_poulsbo"
     fi
 
     PKG_CHECK_MODULES(XEPHYR, $XEPHYR_REQUIRED_LIBS, [xephyr="yes"], [xephyr="no"])
